source/nodejs/adaptivecards/src/card-elements.ts

```diff
@@ -2879,7 +2885,7 @@ export class Container extends CardElement {
         for (let item of this._items) {
             handleElement(item);
         }
-
+        
```

Comment:
Nit: unnecessary whitespace

Reply:
This is followed by "return true". There should always be a blank line after a closing curly bracket unless there is another closing curly bracket on the next line or if there is an else statement.

Reply:
Oh I was referring to the extra spaces that got added to the blank line (darker green highlighted section), not the line itself.

Reply:
Oh! Sorry :)